src/Components/Todo/IndividualTodo.jsx

```diff
@@ -52,7 +52,7 @@ const IndividualTodo = () => {
         time: end,
       };
     });
-    console.log("Check Time", checkTime);
+    //console.log("Check Time", checkTime);
```

Comment:
sob input fild ka 1 ta Component kora use korban @isnamdnazim 

and ? use

Reply:
okay copy

Reply:
Done